packages/web/src/components/Grid/README.md

```diff
@@ -0,0 +1,19 @@
+# Grid
+
+Use Grid to build multiple column layouts. This Grid works on twelve column
+system, and it contains variants of 12, 6, 4, 3, 2, and 1 column for each breakpoint.
+And additionally one centered narrow column. Inner gaps work well with Container padding.
+
+```html
+<div class="Grid Grid--cols-2 Grid--tablet--cols-3 Grid--desktop--cols-4">
```

Comment:
This is a nice example of responsive columns.

Can you please showcase also all equal column variants so one can see what's available? I mean `Grid--cols-1`, `Grid--cols-2`, `Grid--cols-3`, etc.